src/quest_manager/models.py

```diff
@@ -46,11 +46,19 @@ def get_icon_url(self):
             return self.icon.url
         else:
             return SiteConfig.get().get_default_icon_url()
+    
+    def current_quests(self):
+        """ Returns a queryset containing every visible quest in this campaign."""
```

Comment:
Not true, it won't return `visible` quests if they are `archived`.  Just say currently available.